osu.Game/Rulesets/Edit/PlayfieldOverlay.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+
+namespace osu.Game.Rulesets.Edit
+{
+    public class PlayfieldOverlay : CompositeDrawable

```

Comment:
Currently the term "overlay" is used for a global overlay display. I propose we call these "Layers" and this one would be the "SelectionLayer" or similar

Reply:
Imo it's too early to decide this naming. If you want the grid to be in the upper layer then it's not going to be just "selection", and if this layer (or the box contained) should handle moving objects then it also isn't just the selection layer then either.

I propose we keep this for now, since this is just structure.

Reply:
If there's nothing in here then just remove it, and add it back when there's something to name it after IMO.

Strongly against using overlay in a different scenario to actually being a global overlay.

Reply:
Removed then.